packages/sanity/src/desk/comments/src/components/pte/comment-input/Editable.tsx

```diff
@@ -97,43 +90,23 @@ export function Editable(props: EditableProps) {
     value,
   } = useCommentInput()
 
+  const cursorElement = useCursorElement({
+    disabled: !mentionsMenuOpen,
+    rootElement: rootElementRef.current,
+  })
+
   const renderPlaceholder = useCallback(
     () => <PlaceholderWrapper>{placeholder}</PlaceholderWrapper>,
     [placeholder],
   )
 
-  useClickOutside(
-    useCallback(() => {
-      if (mentionsMenuOpen) {
-        closeMentions()
-      }
-    }, [closeMentions, mentionsMenuOpen]),
-    [popoverElement],
-  )
-
-  // Update the mentions search term in the mentions menu
-  useEffect(() => {
-    mentionsMenuRef.current?.setSearchTerm(mentionsSearchTerm)
-  }, [mentionsSearchTerm])
-
-  // Close mentions if the user selects text
-  useEffect(() => {
-    if (mentionsMenuOpen && selection && !isEqual(selection.anchor, selection.focus)) {
-      closeMentions()
-    }
-  }, [mentionsMenuOpen, closeMentions, selection])
-
-  // Close mentions if the menu itself has focus and user press Escape
-  useGlobalKeyDown((event) => {
```

Comment:
This `useGlobalKeyDown` is not necessary since we will close the mentions menu in `handleKeyDown` (case 'Escape').